pkg/deploy/spindeploy/spindeploy.go

```diff
@@ -122,7 +122,14 @@ func (d *Deployer) Deploy(ctx context.Context, svc interfaces.SpinnakerService,
 		}
 	}
 
-	if err = d.deployConfig(ctx, scheme, l, rLogger); err != nil {
+	sUnmanaged := nSvc.GetOperatorConfig().UnmanagedServices
```

Comment:
When the reconcile cycle identify a change, it gets the list of services that won't be managed by operator. create a map with the list of name services